docs/docs/crafty.md

```diff
@@ -3,4 +3,184 @@ id: crafty
+
+```sh
+zos init Crafty
+```
+
+This will create a `zos.json` file, which will store how your project is structured in ZeppelinOS: you'll probably want to commit this file.
+
+### Making `Crafty` upgradeable
+
+There are multiple reasons why we would want to be able to upgrade `Crafty`'s game logic contract: from bug-fixing, to adding new functionalities. Remember how `Crafty` uses RBAC (role-based access control)? An upgrade could easily add new roles, such as a `curator` role, which would be in charge of approving new tokens before they are added to the game, or highlighting featured creations to be displayed on the front page. These simple but useful extensions would not be possible without ZeppelinOS.
+
+`Crafty` is a great example of how easy it is to add upgradeability to your project. It extends OpenZeppelin contracts (the `RBAC` contract), which are safe to use with ZeppelinOS, so the only change that we need to make is a minor one: the constructor must be removed, and its code added to an `initialize` function:
+
+```
+import 'zos-lib/contracts/migrations/Initializable.sol';
+
+contract Crafty is Initializable {
+  // Standard Solidity constructor - will be removed from source code
+  function Crafty() public {
+    addRole(msg.sender, ROLE_ADMIN);
+  }
+
+  // Initializer for integration with ZeppelinOS
+  function initialize(address _initialAdmin) isInitializer public {
+    addRole(_initialAdmin, ROLE_ADMIN);
+  }
+  ...
+}
+```
+
+The `isInitializer` modifier will make sure your `initialize` method is only called once in the whole lifetime of your contract. But you must remember to explicitly call it, since unlike constructors, `initialize` methods are not automatically called at contract creation.
+
+With that change in place, we're all set to start using upgradeable instances of the `Crafty` contract! First, we `add` it to the project's contracts, and then `push` the compiled bytecode of the current version to the blockchain.
+
+```sh
+zos add Crafty
+zos push --network ropsten
+```
+
+We can now `create` new upgradeable instances, all of which will use the same deployed bytecode, but be entirely independent otherwise. The `initialize` function can be called in this same step, saving you the need to do it manually from a console.
+
+```sh
+zos create Crafty --init --args 0x0cbd7... --network ropsten
+> 0x31C4B...
+```
+
+The returned value is the address of the newly `create`d Crafty proxy, which is already initialized and can be safely used, with the peace of mind that it can later be upgraded at any point in time.
+
+### Making `CraftableToken` upgradeable
+
+This scenario is a bit more complex, since multiple ZeppelinOS contracts are being combined together, but it is nonetheless fairly easy to setup. Let's first recap what a `CraftableToken` is:
+
+1. It is detailed ERC20 token (i.e. has name and symbol)
```

Comment:
Numbering typo here?

Reply:
Markdown automatically makes numbered lists when items start with 1.
```
1. this
1. produces
1. that
```
1. this
1. produces
1. that